api/lib/opentelemetry/instrumentation/base.rb

```diff
@@ -225,6 +239,39 @@ def enabled?(config = nil)
 
       private
 
+      # The config_options method is responsible for validating that the user supplied
+      # config hash is valid.
+      # Unknown configuration keys are not included in the final config hash.
```

Comment:
Should we log unknown configuration keys?

Reply:
Yeah I think so, it seems like a helpful addition to anyone getting setup with an instrumentation library.